app/main.py

```diff
@@ -1,6 +1,27 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.getenv("API_KEY")
```

Comment:
Please change the method from optional to strict, as it should raise a KeyError if the key is not provided